suzieq/cli/sqcmds/NamespaceCmd.py

```diff
@@ -0,0 +1,59 @@
+import time
+from nubia import command
+from suzieq.cli.nubia_patch import argument
+
+from suzieq.cli.sqcmds.command import SqTableCommand
+from suzieq.sqobjects.namespace import NamespaceObj
+
+
+@command("namespace", help="Summarize network-wide data")
+class NamespaceCmd(SqTableCommand):
+    """Overall network information such as namespaces present etc."""
+
+    def __init__(
+            self,
+            engine: str = "",
+            hostname: str = "",
+            start_time: str = "",
+            end_time: str = "",
+            view: str = "",
+            namespace: str = "",
+            format: str = "",  # pylint: disable=redefined-builtin
+            query_str: str = " ",
+            columns: str = "default",
+    ) -> None:
+        super().__init__(
+            engine=engine,
+            hostname=hostname,
+            start_time=start_time,
+            end_time=end_time,
+            view=view,
+            namespace=namespace,
+            columns=columns,
+            format=format,
+            query_str=query_str,
+            sqobj=NamespaceObj,
+        )
+
+    @command("show", help="Show device information")
+    @argument("model", description="model(s), space separated")
+    @argument("os", description='NOS(es), space separated')
+    @argument('vendor', description='Vendor(s), space separated')
+    @argument('version', description='NOS version(s), space separated')
```

Comment:
I think this way they are more explanatory
```suggestion
    @command("show", help="Show namespace information")
    @argument("model", description="Device model(s), space separated")
    @argument("os", description='Device NOS(es), space separated')
    @argument('vendor', description='Device vendor(s), space separated')
    @argument('version', description='Device NOS version(s), space separated')
```